### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about rails API

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @azy64 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ]  Create also a pull request in your frontend project that explains what you did.

- [ ] Kindly use react hooks to fetch and dispatch an action when the greeting component is rendered in your frontend project. Right now no message is displayed because no action was dispatch when your page loaded.

image

- [ ] Also remove these commented code here in your frontend part.

image

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by azy64** [comment]
> ## Hi @azy64
> Good job so far! I know you have worked hard on making this milestone But There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> almost-there
> 
> ## To highlight 
> * All linters are passing heavy_check_mark
> * Good commit history heavy_check_mark
> * Good PR description heavy_check_mark
> 
> ## Required Changes 
> * [ ]   Create also a pull request in your frontend project that explains what you did.
> * [ ]  Kindly use react hooks to fetch and dispatch an action when the greeting component is rendered in your frontend project. Right now no message is displayed because no action was dispatch when your page loaded.
> 
> image
> 
> * [ ]  Also remove these commented code here in your frontend part.
> 
> image
> 
> * [ ]  Kindly check comments under the review.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

You should run the server-side as follow: rails s -p 3001

**Comment by azy64** [comment]
the App is working well!!!! the react app running on port 3000 and rails API should run on port 3001. Run the server-side as follow: rails s -p 3001

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-# end
+ Rails.application.config.middleware.insert_before 0, Rack::Cors do
+   allow do
+     origins "localhost:3000"
+
+     resource "*",
+       headers: :any,
+       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+   end
+ end
```
Comment (DeliceLydia) [inline config/initializers/cors.rb]: - [ ] Address what the previous reviewer mentioned I am not able to retrieve greetings on the screen
- [ ] Please open a PR on your frontend repo

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @azy64, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/initializers/cors.rb (hunk 1 above)
Comment (azy64) [reply config/initializers/cors.rb]: did you create a message in the Database?

Comment (azy64) [reply config/initializers/cors.rb]: before all, you should create messages in the database

Comment (azy64) [reply config/initializers/cors.rb]: Please read the project requirement

**Comment by DeliceLydia** [comment]
Yes I created them using rails console

**Comment by azy64** [comment]
did you run the server with the following command: rails s -p 3001

**Comment by azy64** [comment]
and the server should be run in the same machine that the front-end app

**Comment by azy64** [comment]
I tried again on my local machine, everything is running normally

**Comment by DeliceLydia** [comment]
@azy64 I was checking what issue you might have and found the issue
check your `messageReducer.js` you are fetching from `http://127.0.0.1:3000/messages` but remember your backend is running on 3001 so it has to be like this please change it and all will be fine:
fect

#### Keep Rocking

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @azy64, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**